api/src/test/java/com/github/eyefloaters/console/api/support/EnabledOperationFilterTest.java

```diff
@@ -0,0 +1,28 @@
+package com.github.eyefloaters.console.api.support;
+
+import jakarta.ws.rs.core.Response.Status;
+
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.junit.QuarkusTest;
+import io.quarkus.test.junit.TestProfile;
+
+import static com.github.eyefloaters.console.test.TestHelper.whenRequesting;
+import static org.hamcrest.Matchers.contains;
+import static org.hamcrest.Matchers.is;
+
+@QuarkusTest
+@TestProfile(OperationFilterTestProfile.class)
+class EnabledOperationFilterTest {
```

Comment:
We should have a test for create topics being enabled.

Reply:
I actually avoided tests that would perform successful Kafka operations with the `OperationFilterTestProfile` because this profile doesn't start a broker behind the API. There are other integration tests that do exercise the createTopic operation though and the `OASModelFilterTest` checks that operations are selectively on/off regardless of the value of `console.read-only`. Let me know your thoughts if you would still like something additional here.

Reply:
OK, that's fine. Maybe add a comment though.

Reply:
Added Javadoc to `EnabledOperationFilterTest` and `OperationFilterTestProfile` about expectations of a working Kafka cluster for future tests.